File: car-rental-api/car-rental/src/main/java/dev/ikoinaris/carrental/entity/Extra.java
package dev.ikoinaris.carrental.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "extras")
public class Extra {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    @Column(name = "name")
    private String name;

    @Column(name = "img_url")
    private String img_url;

    @Column(name = "dailyPrice")
    private int dailyPrice;
    
}

File: car-rental-ui/src/app/models/extra.ts
export class Extra {
  id: number;
  name: string;
  imgUrl: string;
  dailyPrice: number;

  constructor(id: number, name: string, imgUrl: string, dailyPrice: number) {
    this.id = id;
    this.name = name;
    this.imgUrl = imgUrl;
    this.dailyPrice = dailyPrice;
  }
}
